hack/manifests/testing/kustomization.yaml

```diff
@@ -1,21 +1,23 @@
-bases:
-- ../base
+---
+apiVersion: kustomize.config.k8s.io/v1beta1
+kind: Kustomization
 resources:
 - ingress.yaml
 - ldap.yaml
 - postgres.yaml
+- ../base
 
 patchesStrategicMerge:
 - parodos-patch.yaml
 
+
 images:
 - name: quay.io/parodos-dev/workflow-service:main
   newTag: test
-
 - name: quay.io/parodos-dev/notification-service:main
   newTag: test
 
 configMapGenerator:
-- name: ldif-config
-  files:
+- files:
   - test-server.ldif
+  name: ldif-config
```

Comment:
I do not really like this change in the name, if more files are in there, it's just hard to know the name of the ConfigMap.

Reply:
@eloycoto 
I moved the name entry back up to its original position.
Not sure if I addressed your comment properly.